### ISSUE TITLE
[BUG] Invalid data found when processing input (FFmpeg post processing)

### ISSUE THREAD

**Issue by CyberPoison** [ytdl-org/youtube-dl#25441]
## Checklist

- [X ] I'm reporting a broken site support issue
- [X ] I've verified that I'm running youtube-dl version **2020.05.08**
- [X ] I've checked that all provided URLs are alive and playable in a browser
- [X ] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [ X] I've searched the bugtracker for similar bug reports including closed ones
- [ X] I've read bugs section in FAQ

## Verbose log

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--cookies', '/root/cookies.txt', '--prefer-ffmpeg', '-f', 'best', '-ciw', '--write-info-json', '--write-thumbnail', '--all-subs', '--add-metadata', '--ignore-errors', '-o', '/mnt/transfer/%(uploader)s/%(title)s .%(ext)s', '-v', 'https://www.youtube.com/channel/UCXuqSBlHAE6Xw-yeJA0Tunw']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2020.05.08
[debug] Python version 3.6.9 (CPython) - Linux-4.15.0-101-generic-x86_64-with-Ubuntu-18.04-bionic
[debug] exe versions: ffmpeg 4.2.2, ffprobe 4.2.2
[debug] Proxy map: {}

s' 'file:/mnt/transfer/Linus Tech Tips/Our server room ACTUALLY Caught Fire Explained .temp.mp4'
ERROR: file:/mnt/transfer/Linus Tech Tips/Our server room ACTUALLY Caught Fire Explained .mp4: Invalid data found when processing input
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/youtube_dl/YoutubeDL.py", line 2065, in post_process
    files_to_delete, info = pp.run(info)
  File "/usr/local/lib/python3.6/dist-packages/youtube_dl/postprocessor/ffmpeg.py", line 498, in run
    self.run_ffmpeg_multiple_files(in_filenames, temp_filename, options)
  File "/usr/local/lib/python3.6/dist-packages/youtube_dl/postprocessor/ffmpeg.py", line 235, in run_ffmpeg_multiple_files
    raise FFmpegPostProcessorError(msg)
youtube_dl.postprocessor.ffmpeg.FFmpegPostProcessorError: file:/mnt/transfer/Linus Tech Tips/Our server room ACTUALLY Caught Fire Explained .mp4: Invalid data found when processing input
```

## Description

Looks like than the ffmpeg is not able to do post processing not sure (I think)

**Comment by GonzRon** [comment]
this is a duplicate of #25717

**Comment by dirkf** [comment]
Fixed in 2021.06.06
